### ISSUE TITLE
Upgrading via machine learning

### ISSUE THREAD

**Issue by sSomename** [Yink/Amadeus#107]
Machine learning has seen some major improvements now which would make sense for usage in an app like this. 

With voice cloning ai she could say much more. And  via ml she could have more things to say and respond to. 

Of course i acknowledge this would make it require more then a phone since phones arent capable devices for ml.

**Comment by AtelierMizumi** [comment]
with the existance of ChatGPT, deepfake, AI art, voice cloning

I HAVE A DREAM !

**Comment by Gibbyace** [comment]
Oh yeah baby, its (computer) science time!

thuanc177 ***@***.***> schrieb am Do., 19. Jan. 2023, 14:50:

**Comment by KilluaGremory** [comment]
Has any progress been made via ChatGPT by any chance?

**Comment by KilluaGremory** [comment]
Has any progress been made via ChatGPT by any chance?

**Comment by White-Anubis** [comment]
Yes! After the update the issue now is solveSent from my iPhoneOn Mar 22, 2023, at 8:33 PM, Killua Gremory ***@***.***> wrote:﻿
Has any progress been made via ChatGPT by any chance?

**Comment by KilluaGremory** [comment]
they have not abandoned the project right?
